cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -436,7 +436,15 @@
 
     public static final String DATA_STORAGE_IMPLEMENTATION = "data.storage.implementation";
     public static final String DATA_STORAGE_NOSQL = "nosql";
-    public static final String DATA_STORAGE_SQL = "postgressql";
+    public static final String DATA_STORAGE_SQL = "postgresql";
+    public static final String DATA_STORAGE_SQL_DRIVER_DIRECTORY = "data.storage.sql.driver.directory";
+    public static final String DATA_STORAGE_SQL_JDBC_DRIVER_NAME = "data.storage.sql.jdbc.driver.name";
+
+    // the jdbc connection related properties should be from cdap-security.xml
+    public static final String DATA_STORAGE_SQL_JDBC_CONNECTION_URL = "data.storage.sql.jdbc.connection.url";
+    public static final String DATA_STORAGE_SQL_USERNAME = "data.storage.sql.username";
+    public static final String DATA_STORAGE_SQL_PASSWORD = "data.storage.sql.password";
+    public static final String DATA_STORAGE_SQL_PROPERTY_PREFIX = "data.storage.sql.property.";
```

Comment:
Same question for SQL properties

Reply:
There are a bunch of properties that user can specify: https://jdbc.postgresql.org/documentation/80/connect.html. And they can also specify the username and password using this, so I prefer leaving it in sConf